docs/introduction/zns.md

```diff
@@ -47,91 +47,89 @@ title="Conventional SSDs and ZNS SSDs internal data placement"/>
+(ZSZE) are not mapped to any physical storage blocks and writes to these blocks
+will result in an error. Therefore, reads within this area are handled
+in the same way as reads to unwritted blocks.
 
-A zone with a zone capacity smaller than the zone size will be transitioned to a
-full condition when the number of written blocks equals the zone capacity.
+A zone with a zone capacity smaller than the zone size will be transitioned to
+zone state Full when the number of written blocks equals the zone capacity.
 
 :::note
 The total namespace capacity reported by a controller is always equal to the
-total number of logical blocks defined by the zones. In other words, this
-reported capacity includes unusable logical blocks of zones with a zone capacity
-lower than the zone size. The usable capacity of the namespace is equal to the
-sum of all zones capacities. This usable capacity is always smaller than the
-reported namespace capacity if the namespace contains zones with a zone capacity
-lower than the zone size.
+total number of logical blocks defined by the zones. In other words, if the
+namespace contains zones with zone capacity less than the zone size, the
+reported capacity includes unusable logical blocks (falling b/w ZCAP & ZSZE) and
+the usable capacity of the namespace is equal to the sum of all zones' capacities.
+In this case, the usable capacity available is smaller than the reported namespace
+capacity.
 :::
 
 ### Active Zones
 
 A controller implementation typically requires the allocation of internal
-resources (e.g. a write buffer) to execute write operations into zones.
-Therefore, limitations on the total amount of resources available to the
-controller may imply a limit on the total number of zones that can be
-simultaneously in the implicit open or explicit open conditions. This potential
-limit on the maximum number of open zones is similarly defined in the ZNS, ZBC,
-and ZAC standards.
-
-The ZNS specification however defines an additional limit on the number of zones
-that can be in the implicit open, explicit open or closed conditions. Any zone
-with such condition is defined as an active zone and correspond to any zone that
-is being written or that has been only partially written. A ZNS SSD may impose a
-limit on the maximum number of zones that can be active. This limit is always
-equal or larger than the limit on the maximum number of open zones.
+resources (e.g. write buffers) to execute write operations into zones.
+Limitations on the total amount of resources available in the controller may
+imply a limit on the total number of zones that can simultaneously be in the
+implicit open or explicit open states. This potential limit on the maximum
+number of open zones is similarly defined in the ZNS, ZBC and ZAC standards.
+
+The ZNS specification defines an additional limit on the number of zones
+that can be in the implicit open, explicit open or closed states. Zones mapped
```

Comment:
zones mapped to state ? That does not correspond to anything technical. "Zones that are in one of these states" would be technically correct.